src/guacd/daemon.c

```diff
@@ -333,27 +319,46 @@ int main(int argc, char* argv[]) {
             guacd_log(GUAC_LOG_ERROR, "Unable to resolve host: %s",
                     gai_strerror(retval));
 
+        /* Get socket */
+        socket_fd = socket(current_address->ai_family, SOCK_STREAM, 0);
+        if (socket_fd < 0) {
+            guacd_log(GUAC_LOG_ERROR, "Error opening socket: %s", strerror(errno));
+
+            /** Unable to get a socket for the resolved address family, try next */
```

Comment:
Looks like an extra asterisk snuck in here. I assume this isn't intended to be a Doxygen-style comment. ;)

Reply:
Thanks for the review.

- Comment style fixed
- Also log socket protocol family on failure to bind (may help in fixing the configuration error)